docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -0,0 +1,311 @@
+[role="xpack"]
+[[security-getting-started]]
+== Getting started with security
+
+In this tutorial, you learn how to secure a cluster by configuring users and 
+roles for use in {es}, {kib}, {ls}, and {metricbeat}. 
+
+[float]
+[[gs-security-prereqs]]
+=== Before you begin
+
+Install and start {es}, {kib}, {ls}, and {metricbeat} as described in 
+<<get-started-elastic-stack>>. 
+
+Launch the {kib} web interface by pointing your browser to port 5601. For 
+example, http://127.0.0.1:5601[http://127.0.0.1:5601].
+
+[float]
+[[gs-security-license]]
+=== Install a trial license
+
+When you installed the products, they applied Basic licenses with no expiration 
+dates. To view your license in {kib}, go to **Management** and click 
+**License Management**.
+
+[role="screenshot"]
+image::images/management-license.png["License management screenshot in Kibana"]
+
+To secure your cluster, you must obtain a license that includes the {security} 
+features. For more information about Elastic license levels, see 
+https://www.elastic.co/subscriptions.
+
+You can start a 30-day trial to try out all of the Platinum features, including 
+{security}. Click **Start trial**.
+
+IMPORTANT: If your cluster has already activated a trial license for the current 
+major version, you cannot start a new trial. For example, if you have already 
+activated a trial for v6.0, you cannot start a new trial until v7.0.
+
+At the end of the trial period, your license reverts to a Basic license. If you 
+want to keep using the extra features, you can purchase a subscription. 
+
+[float]
+[[gs-security-enabled]]
+=== Enable security in {es}
+
+When you use the trial license, {security} is disabled by default. You must 
+stop {kib}, stop {es}, then add the `xpack.security.enabled` setting to the 
+`config/elasticsearch.yml` file in your {es} directory. For example:
+
+[source,yaml]
+----
+xpack.security.enabled: true
+----
+
+NOTE: This tutorial involves a single node cluster, but if you had multiple 
+nodes, you would enable {security} on every node in the cluster. If you have a 
```

Comment:
maybe `you would enable {security} on every node in the cluster and configure Transport Layer Security (TLS) for 
+internode-communication, which is beyond the scope of this tutorial.`